README.md

```diff
@@ -1,4 +1,76 @@
 # TeamCity Build Opentelemetry Exporter Plugin
 
-A custom TeamCity plugin that contains a build listener to send OpenTelemetry trace data to an external endpoint
+A custom TeamCity plugin that registers a build listener to send build trace data to an opentelemetry collector external endpoint.
+
+## What is OpenTelemetry
+
+You can use OpenTelemetry to instrument, generate, collect, and export telemetry data (metrics, logs, and traces) for analysis in order to understand your software's performance and behavior. Create and collect telemetry data from your services and software, then forward them to a variety of analysis tools. For more information about OpenTelemetry go to https://opentelemetry.io/. Documentation for instrumentation for Java and other languages can be found [here](https://opentelemetry.io/docs/).
+
```

Comment:
Add another section in here about what this gives us for builds (ie, why should i install it, what can I do with it). Maybe a picture too?

Reply:
Added: 
"Example: A TeamCity composite build with various build dependancies and associated build steps with OpenTelemetry data represented in Honeycomb Traces." (image below)